### PR TITLE
Add missing MarketplaceOrdering operations to swagger

### PR DESCRIPTION
**Pull request by zachraMSFT**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by sergey-shandar** [comment]
@zachraMSFT could you update your branch (merge the latest changes from master)?

**Comment by zachraMSFT** [comment]
> @zachraMSFT could you update your branch (merge the latest changes from master)?

Did that and added the examples to resolve the failing auto checks

File: specification/marketplaceordering/resource-manager/Microsoft.MarketplaceOrdering/stable/2015-06-01/Agreements.json [hunk 1]
```diff
@@ -155,6 +155,156 @@
...
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MarketplaceOrdering/agreements/{publisherId}/offers/{offerId}/plans/{planId}": {
+      "get": {
+        "operationId": "MarketplaceAgreements_GetAgreement",
+        "description": "Get marketplace agreements.",
```
[the remark this answers is not in the document]
Comment (daschult) [reply specification/marketplaceordering/resource-manager/Microsoft.MarketplaceOrdering/stable/2015-06-01/Agreements.json]:
```suggestion
        "description": "Get marketplace agreement.",
```

File: specification/marketplaceordering/resource-manager/Microsoft.MarketplaceOrdering/stable/2015-06-01/Agreements.json [hunk 2]
```diff
@@ -155,6 +155,156 @@
...
+              "$ref": "#/definitions/AgreementTerms"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MarketplaceOrdering/agreements": {
+      "get": {
+        "operationId": "MarketplaceAgreements_List",
+        "description": "List marketplace agreements.",
```
Comment (daschult) [inline specification/marketplaceordering/resource-manager/Microsoft.MarketplaceOrdering/stable/2015-06-01/Agreements.json]:
```suggestion
        "description": "List marketplace agreements in the subscription.",
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by zachraMSFT** [comment]
@daschult It looks like the checks failed due to some internal failure (IP not accessible). Is there something I should do to resolve that?

**Comment by daschult** [comment]
Closing and reopening to restart CI.

**Review by ghost — APPROVED** [review]

**Pull request merged**